src/sql/workbench/services/notebook/browser/sql/sqlSessionManager.ts

```diff
@@ -527,6 +527,13 @@ export class SQLFuture extends Disposable implements FutureInternal {
 			this._dataToSaveMap.set(key, data);
 			this._rowsMap.set(key, []);
 			this.sendIOPubMessage(data, set);
+			// When there is no streaming, the results are returned in one set and
```

Comment:
So I'm not sure we need this comment here - it makes sense to me in general that if we have any rows in this result set that we process them immediately with handleResultSetUpdate. Streaming sending 0 rows in the initial set is 

A. Not something I think we can guarantee. Some providers might return the initial set with some rows I would think?
B. Just an implementation detail regardless

So I'd just make the comment more generic about handling the initial set of rows that come in a result set.

Reply:
Makes sense, I'll update the comment.